vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSlideController.java

```diff
@@ -119,7 +119,9 @@ public String slide(Model model, @PathVariable("slideId") String slideId, @PathV
         if(sequenceHistory.hasHistory()) {
             model.addAttribute("showBackToPreviousChoice", true);
             model.addAttribute("previousSequenceId", sequenceHistory.peekSequenceId());
-            model.addAttribute("previousBranchingPointId", sequenceHistory.peekBranchingPointId());
+            String previousBranchPointId = sequenceHistory.peekBranchingPointId();
+            model.addAttribute("previousBranchingPointId", previousBranchPointId);
+            model.addAttribute("previousBranchingPointName", ((BranchingPoint)slideManager.getSlide(previousBranchPointId)).getName());
```

Comment:
just put the branching point in the model and then use it in the jsp otherwise if we later want to add more properties of the branching point to the page, we'll always have to to change two files the controller and the jsp (instead of just the jsp).